docs/reference/index.rst

```diff
@@ -10,12 +10,13 @@ Python API reference.
    :maxdepth: 1
    :caption: Core utilities
 
+   accounts
+   api_reference
    checks
    context_processors
    migration_operations
-   views
-   api_reference
    templatetags
+   views
```

Comment:
I see you order this alphabetically, but the ordering of these modules *is* deliberate and semantically. The top entries are most likely to be needed, while the bottom entries are less likely.